hazelcast/src/main/java/com/hazelcast/internal/util/JVMUtil.java

```diff
@@ -112,6 +113,19 @@ public static long usedMemory(final Runtime runtime) {
         return used;
     }
 
+    /**
+     * Explicit cast to {@link Buffer} parent buffer type. It resolves issues the covariant return types in Java 9+ for
```

Comment:
Could we be more specific what the exact issue is, or maybe with some link.

Reply:
Will extend in the new PR.